kubernetes/helm_charts/secor/config/secor.partition.properties

```diff
@@ -23,7 +25,7 @@ secor.kafka.group={{ get (get $.Values.secor_jobs $.Release.Name) "consumer_grou
 secor.message.parser.class={{ get (get $.Values.secor_jobs $.Release.Name) "message_parser" }}
 
 # S3 path where sequence files are stored.
-secor.s3.path=
+secor.s3.path={{- get (get $.Values.secor_jobs $.Release.Name) "base_path" }}
```

Comment:
@ddevadat secor.s3.path also needs to be moved to secor.s3.properties file and configured in the s3 related config file.

Reply:
its restructured now